src/lit/image.ts

```diff
@@ -110,7 +110,8 @@ export class DeguImage extends LitElement {
     this.isFife =
```

Comment:
Update this varname too, e.g. `this.isGoogleImage`

Reply:
Oops.. Thank you.  Done